app/mailers/waste_exemptions_engine/confirmation_mailer.rb

```diff
@@ -42,5 +43,12 @@ def generate_pdf_certificate
       nil
     end
 
+    def privacy_policy
+      root_path = File.join(__dir__, "..", "..", "..")
+      privacy_pdf_path = File.absolute_path(File.join(root_path, "lib/privacy_policy.pdf"))
```

Comment:
What about using `Rails.root.join("lib/privacy_policy.pdf")`? I'm not 100% sure it will work on Rails 4 but it should :) Worth testing it :D

Reply:
As it's an engine and not in a rails app directly I didn't think to try it. But as this engine will only ever be used in a rails app I will!

Reply:
☹️ No go. `Rails.root` works, but it's not where I need to be. The pdf is in the engine (Gem's) `lib/` folder and as such `Rails.root` puts me in the wrong place.

For example in our vagrant environment when running the tests `Rails.root.join("lib/privacy_policy.pdf")` becomes `/vagrant/waste-exemptions-engine/spec/dummy/lib/privacy_policy.pdf`. I need it to be `/vagrant/waste-exemptions-engine/lib/privacy_policy.pdf`.

Reply:
Oh! Well, this is good to know for the future then :D